packages/cli/src/server/util/debugger-ui/debuggerWorker.js

```diff
@@ -18,11 +18,11 @@ onmessage = (function() {
     var hasWarned = false;
     return function() {
       // Wait until `YellowBox` gets initialized before displaying the warning.
-      if (hasWarned || console.warn.toString().includes('[native code]')) {
+      if (hasWarned || logger.warn.toString().includes('[native code]')) {
         return;
       }
       hasWarned = true;
-      console.warn(
+      logger.warn(
```

Comment:
we don't want logger in `debugger-ui` because it runs in a browser

Reply:
fixed